exampleCourse/questions/element/hiddenHints/question.html

```diff
@@ -40,4 +56,4 @@
       This is a hint that will be shown only on the submission panel.
     </pl-hint>
   </pl-hidden-hints>
-</pl-submission-panel>
+</pl-submission-panel>
```

Comment:
Please make sure to not remove the newline at the end of the file. If you open PrairieLearn as a project in VSCode, it shouldn't do this.